packages/peregrine/lib/talons/SignIn/useSignIn.js

```diff
@@ -36,7 +36,8 @@ export const useSignIn = props => {
         errors.push(getDetailsError);
     }
 
-    const formRef = useRef(null);
+    const formApiRef = useRef(null);
+    const setFormApi = useCallback(api => (formApiRef.current = api), []);
```

Comment:
We should not memoize callbacks that do not change; I believe this is actually more costly than `api => (formApiRef.current = api)`.

Reply:
Oops! Just a habit now, though I really doubt it's _that_ costly :P